src/controllers/register-create-password-controller.ts

```diff
@@ -73,9 +73,12 @@ export function createAccountPost(
           req.body["password"]
         )
       ) {
-        return res.redirect(PATH_NAMES.CREATE_ACCOUNT_ENTER_PHONE_NUMBER);
+        if (req.session.user.email.includes("no2fa")) {
```

Comment:
is this not a property that comes back from the web request? This on the email in the session